projects/dotcms-field-elements/src/components/dot-form/dot-form.tsx

```diff
@@ -79,11 +78,18 @@ export class DotFormComponent {
     render() {
         return (
             <form onSubmit={this.handleSubmit.bind(this)}>
-                <div class="form__fields">
-                    {this.fields.map((field: DotCMSContentTypeField) => this.getField(field))}
-                    <slot />
-                </div>
-
+                {this.fields.map((row: DotCMSFieldRow) => {
+                    return (
+                        <div class="form__row">
+                            {row.columns.map((fieldColumn: DotCMSFieldColumn) => {
```

Comment:
This needs abstraction, you should create a row component that receive a one or may columns and a columns component that recieve one or many field.